lib/core/closeDatabaseConnection.js

```diff
@@ -0,0 +1,11 @@
+var debug = require('debug')('keystone:core:closeDatabaseConnection');
+
+module.exports = function closeDatabaseConnection (callback) {
+	this.closeDatabaseConnection = function () {
+		this.mongoose.disconnect(() => {
```

Comment:
We're still targeting Node 0.12 which means vanilla ES5 on the server-side stuff. We'll switch to ES6 for node 4 (LTS) shortly after that's out.